.pre-commit-hooks.yaml

```diff
@@ -1,16 +1,16 @@
 - id: gitleaks
   name: Detect hardcoded secrets
   description: Detect hardcoded secrets using Gitleaks
-  entry: gitleaks protect --verbose --redact --staged
```

Comment:
@adam-moss thanks for the PR. I believe the `--verbose` and `--redact` flags should be present still.

Reply:
I can add them for explicitness if you prefer, but `--redact` would be the same as `--redact=100` which is the default I think?

Reply:
Updated @zricethezav